tests/__init__.py

```diff
@@ -110,8 +113,15 @@ def test_func(self):
                         msg=f"The actual value for .{key}() did not match the expected value.",
                     )
 
-        # Assert that the ingredients returned by the ingredient_groups() function
-        # are the same as the ingredients return by the ingredients() function.
+        # Special handling for ratings_count
+        if "ratings_count" in expect.keys():
+            with self.subTest("ratings_count"):
+                ratings_count = actual.ratings_count()
+                self.assertTrue(
+                    ratings_count >= expect["ratings_count"],
+                    msg=f"Expected ratings_count() : {ratings_count} ",
+                )
```

Comment:
Even though I often find it hard to remember the argument order, we could allow the standard library to construct the messaging for us here:

```suggestion
                self.assertGreaterEqual(
                    ratings_count,
                    expect["ratings_count"]
                )
```

Reply:
Didn't even know this was a possibility , i'll have to read more about the assert functions